ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallDialog.kt

```diff
@@ -38,24 +38,26 @@ fun PaywallDialog(
         }
     }
     if (shouldDisplayDialog) {
```

Comment:
I'm thinking, since we are going to be handling the visibility ourselves, we should remember this parameter through configuration changes. For example, when going between portrait/landscape, changing dark mode or locale,...

I haven't tested it but if I'm not wrong, currently, if we display the paywall, the user dismisses it, and then there is one of those configuration changes, it would reappear. I think we probably want to avoid that by making that state survive configuration changes. One of the options is to use [rememberSaveable](https://developer.android.com/jetpack/compose/state#state-in-composables), instead of `remember`, that's probably the quickest. The other would be moving this state to a viewModel, which might be harder in this case since we don't have a viewModel for the dialog currently.

Reply:
That's a really good point. I just read about the difference and that makes sense 👍🏻 
I just changed this.